packages/web/server/addToCRM.ts

```diff
@@ -153,9 +153,9 @@ export default async function addToCRM({
     const contact = await upsertContact(preparedContact)
     const contactID = Number(contact.contact.id)
 
-    addToList({ contact: contactID, list, status: ListStatus.subscribe })
+    await addToList({ contact: contactID, list, status: ListStatus.subscribe })
```

Comment:
Technically these could be a 
```
await Promise.all([
addToList({ contact: contactID, list, status: ListStatus.subscribe }),
setCustomFields(contactID, { interest, company, role, source: 'website' })
])

```

Reply:
Are both of them slow? If not, the complexity is not worth the upside.

Reply:
they both depend on a third party HTTP API so slower than some things.

Reply:
🤷🏻‍♂️

Reply:
Fixed.